### ISSUE TITLE
Applying Shader cosmetics to Immersive Engineering items turns them white.

### ISSUE THREAD

**Issue by FlameyFaucet** [spoorn/sodium-forge#181]
### Expected Behavior
The items that are compatible with IE's shaders should display the shader's additional colors when applied.

### Actual Behavior
They do not display any color, and the resulting item will show a grey/white model. Any patterns such as stripes are retained and colored, and revolvers appear to be immune to this issue.

### Reproduction Steps
1. Get an IE shader by opening a grab-bag, rarity does not matter.
2. Get an Engineer's workbench and apply the shader to any of the items: The Chemthrower, Plated Shield, and Railgun
3. Tada, white model. If on the shield, the pattern will still be noticeable.

### Mods list
This happens with just Sodium and Immersive Engineering.

### System Information

- Java Version: 1.8.0
- CPU: AMD Athlon 300U
- GPU: AMD Radeon Vega 3 Graphics
- Sodium Reforged Version: 2.9.10

A fix for this is by adding mixin.features.item=false to the mixin config (taken from a comment in the Tinker's Construct color issue). However, this also results in items with animated textures not doing the animations.

**Comment by Cu1sans** [comment]
I think that this happens with tinker's construct as well, though without shaders. When I remove sodium reforged it goes away.

**Comment by FlameyFaucet** [comment]
Yeah. I also have an issue on that too- https://github.com/spoorn/sodium-forge/issues/163
